test/test-applications/integrations/TestApplication.Modules/Program.cs

```diff
@@ -0,0 +1,50 @@
+// <copyright file="Program.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using TestApplication.Shared;
+
+namespace TestApplication.Smoke;
+
+public class Program
+{
+    public const string SourceName = "MyCompany.MyProduct.MyLibrary";
```

Comment:
not used

Reply:
Removed. https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/1515/commits/42aebf63e7b2b5c18f66335bfb97729405d00203